src/main/java/Task.java

```diff
@@ -0,0 +1,55 @@
+public class Task {
+
+    protected String description;
+    protected String eventDate;
+    protected boolean isDone;
+
+    enum Level {
+        low,
+        average,
+        high
```

Comment:
I think that constants in enum should be in all uppercase, eg "LOW", "AVERAGE" and "HIGH"